about.html

```diff
@@ -0,0 +1,123 @@
+        <div class="menu" id="menu">
+            <ul class="list-group">
+              <li><i class="fa fa-close" onclick="hide();"></i></li>
+              <li><a href="index.html"><h3>Home</h3></a></li>
+                <li><hr> </li>
+              <li><h3>Blog</h3></li>
+              <li><hr></li>
+              <li><h3>Contact</h3></li>
+              <li><hr></li>
+              <li><h3>Help</h3></li>
+              <li><hr></li>
+              <li><h3>News</h3></li>
+            </ul>
+          </div>   
+          <div class="topheader">
+            <ul class="list-inline">
+             <li class="list-inline-item"><i class="fa fa-facebook"></i></li>
+             <li class="list-inline-item"><i class="fa fa-twitter"></i></li>
+             <li class="list-inline-item">English</li>
+             <li class="list-inline-item">My page</li>
+             <li class="list-inline-item">Logout</li>
+            </ul>
+     </div>    
+        <nav class="navbar">
+         <i class=" fa fa-solid fa-bars" id="humBtn" onclick="show();"></i>
+        <a href="index.html"> <img src="./image/logo2.svg" alt="logo"></a>
+         <ul class="list-inline">
+          <li class="list-inline-item"><a href="about.html"><b>About</b></a></li>
+          <li class="list-inline-item">Blog</li>
+          <li class="list-inline-item">Contact</li>
+          <li class="list-inline-item">Help</li>
+          <li class="list-inline-item">News</li>
+          <li class="list-inline-item title">Open Library</li>
+         </ul>
+        </nav>
+        <section class="home-content">
+            <div class="content about">
+                <h5>"Hello! Book Lovers"</h5>
+                <h3>
+                    There are also twenty public libraries.
+                </h3>
+                <p class="card">
+                    Welcome to Faena Aleph, an editorial pulse which, 
+                    we warn you, will try to transform you. We are 
+                    a digital magazine that acts as an agent of that 
+                    precise, perhaps omnipresent, moment that triggers inspiration.
+                    Here we perceive ourselves as a garden of shared imagination, an opportunity 
+                    to honor the values that identify us: creativity, rebellion, collaboration, 
+                    magic, eccentricity and transformation
+                </p>
+                <h4>Several free libraries are maintained.</h4>
+                <h6>Contact us via E-mail</h6>
+                <h6>World.Library@gmail.com</h6>
+                <h2>Contact us via E-mail :World.Library@gmail.com</h2>
+            </div>
+        </section>
+        <section class="main-about">
+           <div class="about-desc">
+            <h5>Free Online Libraries You Should Know About</h5>
+            <hr>
+            <p>There are more than 129,864,880 books in the world.
+                Here we perceive ourselves as a garden of shared imagination, an opportunity 
```

Comment:
- [ ] This section looks too squished together. Please apply some margins and try to make it look more like the Behance design 👍 
### Your Design
![image](https://user-images.githubusercontent.com/46839548/175609491-8906e2a5-eaa8-47df-af79-ae4fa12e3d6a.png)
### Behance Design
![image](https://user-images.githubusercontent.com/46839548/175609724-2631962a-2689-4d46-a6f6-75e6e1758a47.png)